modelforge/train/training.py

```diff
@@ -44,6 +44,14 @@
 ]
 
 
+def _exchange_per_atom_energy_to_per_molecule_energy(prop: str):
```

Comment:
super minor here, but this should be either `_exchange_per_atom_energy_for_per_molecule_energy` or `_change_per_atom_energy_to_per_molecule_energy`

Reply:
thanks :-) adopted your first suggestion